_posts/2024-07-23-jenkins-pipeline.md

```diff
@@ -0,0 +1,129 @@
+---
+layout: post
+title:  "[Jenkins] Jenkins Pipeline"
+date:   2024-07-23
+author: Marcelo Mendes Spessoto Junior
+author_website: https://marcelospessoto.github.io/
+tags: [jenkins, ci]
+---
+
+# Exploring Jenkins - Part 2
+
+In this post we are going to understand the Jenkins Job with a pratical implementation.
+
+## Running the Jenkins Docker image
+
+Instead of installing and setting up a Jenkins environment on our host device, let's play with it on a 
+Docker image first. There is a [Docker image specific for running Jenkins already on Docker Hub](https://hub.docker.com/r/jenkins/jenkins).
+
+Let's pull the image with `docker pull jenkins/jenkins:lts-jdk17` and then run it with 
+`docker run -p 8080:8080 jenkins/jenkins:lts-jdk17`.
+
+We can now access the web interface from `localhost:8080`.
+
+It will redirect us to the Jenkins installation. Copy the password printed on the terminal and paste it
+in the required field of the web interface. Then choose to install recommended plugins.
+
+## Running our first Job in Jenkins
+
+Since we are just testing for now, we can skip the **highly recommended step of setting up a Jenkins agent**.
+
+On the Jenkins Dashboard, select the "Create a job" button. For this tutorial, we are going for 
+"Freestyle project".
+
+![Image]({{site.url}}/images/jenkins-pipeline/Jenkins_dashboard.png)
```

Comment:
I've noticed some images aren't rendering, as seen in the print below. Your first post also has an image that isn't being rendered. I don't know if it is a problem on my end when running the blog in localhost, though, so check if you see this, too.

![2024-08-22-080812_938x1026_scrot](https://github.com/user-attachments/assets/c04d4be9-38ad-49ae-87d9-b51b44a19bd7)